contracts/zxp/game/seasons/mechanics/StakerRewards.sol

```diff
@@ -58,11 +58,14 @@ contract StakerRewards is ObjectRegistryClient, IStakerRewards {
             numBlocks = block.number - stakedAt;
         }
         claimedAt[id] = block.number;
+        // TODO should this include a zero check so we don't call unnecessarily?

```

Comment:
Maybe, but that also unnecessarily checks for 0 every time then, where it should usually not happen that the value is actually 0. 
but, checking 0 is a common approach.